content/en/docs/reference/command-line-tools-reference/kube-controller-manager.md

```diff
@@ -439,7 +439,7 @@ kube-controller-manager [flags]
 <td colspan="2">--feature-gates mapStringBool</td>
 </tr>
 <tr>
```

Comment:
This file is auto-generated, will be refreshed when the new release is out.

Reply:
Fixed